pkg/static/assets_generate.go

```diff
@@ -14,7 +14,7 @@ func main() {
 		PackageName:  "static",
 		BuildTags:    "!dev",
 		VariableName: "Assets",
-		Filename:     "static/generated_assets.go",
+		Filename:     "pkg/static/generated_assets.go",
```

Comment:
This needs to be 
```
Filename:     "../static/generated_assets.go",
```
as this contextualized from the source position. There may be also some changes to this when moving `main.go` into `pkg`.